### ISSUE TITLE
Add CI performance testing

### ISSUE THREAD

**Issue by brendt** [tempestphp/tempest-framework#562]
I think it makes sense to have some kind of performance testing/benchmark where we benchmark a Tempest app as if it was running in production (we could use tempest/docs) for it for now.

Maybe this CI action shouldn't fail if performance decreased, but it should give an indication of how PRs impact framework performance (similar to what we do with coveralls for test coverage). 

@aidan-casey @Treggats maybe you have some input here?

**Comment by aidan-casey** [comment]
Yeah, this should be pretty easy to do. As we look at this, I think it probably makes sense to revisit the benchmark app and make it easier to setup.

**Comment by innocenzi** [comment]
Should we close this? We have a few benchmarks now, but not on a full app

**Comment by brendt** [comment]
We can close for now and improve the benchmarks as we go
